inapppurchase/src/main/java/com/tealium/inapppurchase/PurchaseListener.kt

```diff
@@ -0,0 +1,47 @@
+package com.tealium.inapppurchase
+
+import com.android.billingclient.api.BillingClient
+import com.android.billingclient.api.BillingResult
+import com.android.billingclient.api.Purchase
+import com.android.billingclient.api.PurchasesUpdatedListener
+import com.tealium.core.Logger
+import com.tealium.core.Tealium
+
+class PurchaseListener: PurchasesUpdatedListener {
+    override fun onPurchasesUpdated(billingResult: BillingResult, purchases: MutableList<Purchase>?) {
+        Tealium.names().forEach { instanceName ->
+            when (billingResult.responseCode) {
+                BillingClient.BillingResponseCode.OK -> {
+                    purchases?.forEach {  purchaseItem ->
+                        when(purchaseItem.purchaseState) {
+                            Purchase.PurchaseState.PURCHASED -> {
+                                Logger.dev(BuildConfig.TAG, "Tracking purchase with order id: ${purchaseItem.orderId}")
+
+                                Tealium[instanceName]?.inAppPurchaseManager?.trackInAppPurchase(purchaseItem)
```

Comment:
Would it be possible to pass the context through here and use that instead of the instance name?